es/cheatsheet-deep-learning.md

```diff
@@ -1,291 +1,291 @@
 
 <br>
 
 **22. Batch normalization ― It is a step of hyperparameter γ,β that normalizes the batch {xi}. By noting μB,σ2B the mean and variance of that we want to correct to the batch, it is done as follows:**
 
-&#10230;
+&#10230; Normalización de grupo - Es un paso de híperparámetro y,β que normaliza el grupo {xi}. Notando μB,σ2B la media y la varianza de qué queremos corregir en el grupo, se realiza de la siguiente manera:
 
 <br>
 
 **23. It is usually done after a fully connected/convolutional layer and before a non-linearity layer and aims at allowing higher learning rates and reducing the strong dependence on initialization.**
 
-&#10230;
+&#10230; Se realiza usualmente después de una capa completamente conectada/convolucional y antes de una capa no-lineal y su objetivo es permitir tasas más altas de aprendizaje y reducir su fuerte dependencia hacia la inicialización.
 
 <br>
 
 **24. Recurrent Neural Networks**
 
-&#10230;
+&#10230;Redes Neuronales Recurrentes
 
 <br>
 
 **25. Types of gates ― Here are the different types of gates that we encounter in a typical recurrent neural network:**
 
-&#10230;
+&#10230; Tipos de compuertas - A continuación, tenemos los diferentes tipos de compuertas que encontramos en una red neuronal recurrente típica:
 
 <br>
 
 **26. [Input gate, forget gate, gate, output gate]**
 
-&#10230;
+&#10230; [Compuerta de entrada, compuerta de olvido, compuerta, compuerta de salida]
 
 <br>
 
 **27. [Write to cell or not?, Erase a cell or not?, How much to write to cell?, How much to reveal cell?]**
 
-&#10230;
+&#10230; [¿Escribir o no en la celda?, ¿Borrar o no la celda?, ¿Cuánto escribir en la celda?, ¿Cuánto revelar la celda?]
 
 <br>
 
 **28. LSTM ― A long short-term memory (LSTM) network is a type of RNN model that avoids the vanishing gradient problem by adding 'forget' gates.**
 
-&#10230;
+&#10230; LSTM - Una red de memoria de corto plazo larga (LSTM por sus siglas en inglés) es un tipo de modelo de red neuronal recurrente que evita el problema del gradiente desvaneciente añadiendo puertas de 'olvido'.
 
 <br>
 
 **29. Reinforcement Learning and Control**
 
-&#10230;
+&#10230; Aprendizaje reforzado y control.
 
 <br>
 
 **30. The goal of reinforcement learning is for an agent to learn how to evolve in an environment.**
 
-&#10230;
+&#10230; El objetivo del aprendizaje reforzado es hacer que un agente aprenda como evolucionar en un ambiente.
 
 <br>
 
 **31. Definitions**
 
-&#10230;
+&#10230; Definiciones
 
 <br>
 
 **32. Markov decision processes ― A Markov decision process (MDP) is a 5-tuple (S,A,{Psa},γ,R) where:**
 
-&#10230;
+&#10230; Procesos decisivos de Markov - Un proceso decisivo de Markov (MDP por sus siglas en inglés) es una 5-tupla (S,A,{Psa},γ,R) donde:
```

Comment:
"_Procesos de decisión de Markov_". "Decisivo" is "decisive", but here the word is "decision". Change it twice.